docs/apache-airflow-providers-common-sql/index.rst

```diff
@@ -18,6 +18,10 @@
 ``apache-airflow-providers-common-sql``
 =======================================
 
+Supported Database Types
```

Comment:
The problem you see now is tha the document should be included in one of the toctrees below, not as link (likely in `References`  - because that seems like a reference document

Reply:
I added the link to the reference, and the error is gone. 

Is it fine to keep this part?
```
Supported Database Types
------------------------

:doc:`Supported Database Types <supported-database-types>`
```

Reply:
Likely better to remove - it will be a link on the left side in "References" so good-enough to find it.